src/Microsoft.DotNet.Wpf/tests/UnitTests/PresentationCore.Tests/PresentationCore.Tests.csproj

```diff
@@ -25,6 +25,8 @@
     <PackageReference Include="FluentAssertions" Version="$(FluentAssertionsVersion)" />
     <PackageReference Include="System.Collections.Concurrent" Version="4.3.0" />
     <PackageReference Include="System.Runtime.Serialization.Formatters" Version="9.0.0-preview.7.24359.11 " />
```

Comment:
This looks wrong.  We shouldn't have hardcoded versions in the projects.

Reply:
I see, that was fixed with https://github.com/dotnet/wpf/commit/fbff6e12cdc0bac3256e171dcd00bb95319b9485

Reply:
I tried to keep the changes to minimum and pick only the changes that were introduced in the 2 original PRs.